tools/core_packages.txt

```diff
@@ -7,6 +7,7 @@ c_ares
 ca_certificates
 crew_profile_base
 curl
+e2fsprogs
```

Comment:
Can we add libunbound also or is it not needed now that curl is a fully static build?

Reply:
I don't know the rationale for putting it there in the first place, but if the reason was "because curl needs it" then use it can be removed...

Reply:
libunbound is not a runtime dependency of curl now that curl is static.

Reply:
Please do a fresh install and compare the base packages with core_packages.txt.  The way I do it is with vimdiff.
```
cd /path/to/tools
crew list installed -d > installed.txt
vimdiff installed.txt core_packages.txt
```
Adjust any differences.